src/pages/en/guides/integrations-guide.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Using Integrations
+---
+
+**Astro Integrations** add new functionality and behaviors for your project with only a few lines of code. You can write a custom integration yourself, or grab popular ones from npm. 
+
+- Unlock React, Vue, Svelte, Solid, and support for other popular UI frameworks.
+- Integrate popular tools like Tailwind, Turbolinks, and Partytown with only a few lines of code.
+- Add new features to your build, like automatic sitemap generation.
+- Write custom hooks into the build process, dev server, and more.
```

Comment:
This line is a bit odd to read - maybe re-write it as something like "write custom code that hooks into the build process, dev server, and more."